<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QWidget" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>5</width>
    <height>5</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генератор заданий</string>
  </property>
  <widget class="QSpinBox" name="spVariantNumber">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spUnderscorePart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>0.900000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="mMemoIn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>365</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="centerOnScroll">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Тип задания:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTestCase">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>330</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1. Расположить буквы в словах в случайном порядке</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Заменить часть букв нижним подчеркиванием</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3. Пропустить определенные буквы в слове</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4. Сопоставить слова</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5. Расположить слова в предложениях в случайном порядке</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6. Расположить предложения в случайном порядке</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="edMissedLetters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>aeiouy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пропустить эти буквы:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Количество варинтов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>220</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Часть пропущенных букв:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="mMemoOut">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>501</width>
     <height>531</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="centerOnScroll">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTestGenerate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>330</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Генерировать задания</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
